utils/ShaderJSONGen.py

```diff
@@ -0,0 +1,26 @@
+import json
+import sys
+
+def gen_program(name="bunny_prog"):
```

Comment:
So this file generates json for the shaders? It doesn't have to be run in order to compile anything, right?